sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusProcessorClient.java

```diff
@@ -169,8 +169,10 @@ public void onNext(ServiceBusMessageContext serviceBusMessageContext) {
                             processMessage.accept(serviceBusReceivedMessageContext);
```

Comment:
Do you need to complete the message if isDisableAutoComplete() == false? This statement is from above,"By default, a successfully processed message is * {@link ServiceBusReceivedMessageContext#complete() completed}"

Reply:
The underlying async receiver will do that automatically because I delegate that to the async client in the builder.